sequencer/src/block/tables.rs

```diff
@@ -44,6 +46,13 @@ pub struct NameSpaceTable<TableWord: TableWordTraits> {
     pub(super) phantom: PhantomData<TableWord>,
 }
 
+impl<TableWord: TableWordTraits> EncodeBytes for NameSpaceTable<TableWord> {
+    fn encode(&self) -> std::sync::Arc<[u8]> {
+        let boxed_slice: Box<[u8]> = self.bytes.clone().into_boxed_slice();
+        Arc::from(boxed_slice)
```

Comment:
Can't just do `Arc::from(self.bytes.clone())`?

Reply:
Ah crap sorry I meant to swap this.